src/components/Contacts/ContactDetails/ContactDontationsTab/ConatctDonationsTab.test.tsx

```diff
@@ -32,11 +33,24 @@ describe('ContactDonationsTab', () => {
     `);
     expect(
       result.current.data?.contact.donations.nodes.length,
-    ).toMatchInlineSnapshot(`1`);
+    ).toMatchInlineSnapshot(`undefined`);
```

Comment:
This is interesting to have the text undefined instead of actually undefined, I'm guessing it is part of the test library?